content/api/commands/mount.md

```diff
@@ -0,0 +1,607 @@
+---
+title: mount
+componentSpecific: true
+---
+
+<Alert type="warning">
+
+Cypress does not have a **cy.mount()** command out-of-the-box. See below for
+info on how to craft your own.
+
+</Alert>
+
+For
+[Component Testing](/guides/overview/choosing-testing-type#What-is-Component-Testing),
+we recommend creating a custom `cy.mount` command which wraps the mount command
```

Comment:
Can you use `cy.mount()` with the `()` throughout this doc? Someone once told me they like using the parens. It signifies its a method off cy and not just a value.